src-vue/src/views/SettingsView.vue

```diff
@@ -46,6 +46,15 @@ export default defineComponent({
         flightcoreVersion(): string {
             return this.$store.state.flightcore_version;
         },
+        enableReleasesSwitch: {
+            get(): boolean {
+                return this.$store.state.enableReleasesSwitch;
+            },
+            set(value: boolean): void {
+                this.$store.state.enableReleasesSwitch = value;
+                persistentStore.set('northstar-releases-switching', { value });
+            }
```

Comment:
When disabling showing other release channel, we also wanna switch back to stable release.

Otherwise one can enable release switching -> switch to release candidate -> disable release switching -> they are now "stuck" on release candidate.

Reply:
We could delay fixing that to post merge though if you wanna ;)